scripts/ddtest

```diff
@@ -40,7 +40,7 @@ if [[ "${CIRCLECI}" = "true" ]]; then
                    --quiet-pull \
                    --rm \
                    testrunner \
-                   bash -c "$FULL_CMD"
+                   bash -c "ulimit -c unlimited || true && $FULL_CMD"
```

Comment:
Do we need to be concerned about any limits set by CircleCI?

Reply:
We are already `ulimit -c unlimited` on the "bare" CircleCI runner, so we shouldn't have any sorts of problems with limits 🤞